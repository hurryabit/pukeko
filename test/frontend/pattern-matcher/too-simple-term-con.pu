-- FIXME: This should not fail in the parser but in the pattern match compiler.
data T a = C a
f :: a -> T a
f x = case C x of
  y -> y
