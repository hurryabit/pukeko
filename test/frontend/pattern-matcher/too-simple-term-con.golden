test/frontend/pattern-matcher/too-simple-term-con.pu:5:3:
  |
5 |   y -> y
  |   ^
unexpected 'y'
expecting data constructor

